main.go

```diff
@@ -52,6 +52,13 @@ func handler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	clusterID = os.Getenv("EKS_CLUSTER_ID")
+	clusterID = values.Get("cluster_id")
```

Comment:
I have the feeling that the env var will be always overridden by the cluster_id set on URL. So I guess line 55 is useless ?